### PR TITLE
Optimizes ZAS Subsystem tick , roach life() tick , sanity view() tick

### PR DESCRIPTION
**Pull request by MLGTASTICa**
## About The Pull Request
Using new revolutionary methods , Eris lag has been cut in half, from the most stressing subsystems . ZAS,  roach life() and sanity tick
## Why It's Good For The Game
Eris 120fps?

## Changelog
:cl:
code:Optimized roach life tick even further
code:Optimized Sanity tick.
code:Optimized ZAS tick
/:cl:

### CODE REVIEW

**Comment by TheShown911** [comment]
How can I approve this, there is no stamina?

**Comment by MLGTASTICa** [comment]
> How can I approve this, there is no stamina?

Stamina is coming next year , Handyman is coding it for me.

File: code/controllers/subsystems/air.dm [hunk 1]
```diff
@@ -84,7 +84,7 @@ SUBSYSTEM_DEF(air)
 /datum/controller/subsystem/air/fire(resumed = 0)
 	var/timer = world.tick_usage
 
-	if (currentpart == SSAIR_PIPENETS || !resumed)
+	if ((currentpart == SSAIR_PIPENETS || !resumed) && prob(30))
```
[the remark this answers is not in the document]
Comment (MLGTASTICa) [reply code/controllers/subsystems/air.dm:87]: Its looping.

**Comment by Firefox13** [comment]
not even funny

**Comment by MLGTASTICa** [comment]
Managing to trigger the host makes it funny

**Pull request closed without merging**